src/internal/packager/git/pull.go

```diff
@@ -27,7 +27,7 @@ func (g *Git) DownloadRepoToTemp(gitURL string) string {
 	// packaged anyway, and it saves us from having to fetch the tags
 	// later if we need them
 
-	err = g.pull(gitURL, path, "")
+	_ = g.pull(gitURL, path, "")
```

Comment:
Can we add a comment as to why we ignore this error

Reply:
I actually dont know why we are.........

Reply:
Yeah I think DownloadRepo to temp is used by our helm chart logic only. We fatalf above, could we fatalf here too?

Reply:
we've been trying to remove all the fatals from the git package actually

Reply:
Then we can refactor it to return an error instead potentially.  Or if we want to do that in another PR add a TODO comment.